andrerferrer/rbnb-734-berlin
# Add navbar to the app layout

**What I did:**

I created the navbar using the LW ui Kit. It looks amazing! the best thing is Seb's Picture.

**How to test it:**
To test it you should run `rails s` and go to the root page. you should see the navbar on the top.
